.github/workflows/broker.yml

```
@@ -206,7 +206,7 @@ jobs:
         uses: docker/build-push-action@v2.10.0
         with:
           context: .
-          file: ./Dockerfile
+          file: ./Dockerfile-broker
```

```suggestion
          file: ./Dockerfile.broker
```

This way editors will recognise syntax highlighting correctly. Same thing for `Dockerfile-tracker`.

---

Renaming Dockerfile breaks `Makefile`!